pvlib/iotools/epw.py

```diff
@@ -0,0 +1,183 @@
+    liquid_precipitation_quantity       The period of accumulation for the liquid precipitation depth field, hour
+    =============================       ======================================================================================================================================================
+
+    References
+    ----------
+
+    [1] EnergyPlus documentation, Auxiliary Programs
+    https://energyplus.net/documentation.
+    '''
+    
+    if filename.startswith('http'):
+        # Attempts to download online EPW file
+        # See comments above for possible online sources
+        request = Request(filename, headers={'User-Agent': (
+            'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_5) '
+            'AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.87 '
+            'Safari/537.36')})
+        response = urlopen(request)
+        csvdata = io.StringIO(response.read().decode(errors='ignore'))
+    else:
+        # Assume it's accessible via the file system
+        csvdata = open(filename, 'r')
+    
+    # Read line with metadata
+    firstline = csvdata.readline()
+
+    head = ['loc','city', 'state-prov', 'country', 'data_type','WMO_code', 
+            'latitude', 'longitude', 'TZ','altitude']
+    meta = dict(zip(head, firstline.rstrip('\n').split(",")))
+
+    meta['altitude'] = float(meta['altitude'])
+    meta['latitude'] = float(meta['latitude'])
+    meta['longitude'] = float(meta['longitude'])
+    meta['TZ'] = float(meta['TZ'])
+
+    colnames = ['year', 'month', 'day', 'hour', 'minute', 'data_source_unct', 
+                'temp_air', 'temp_dew', 'relative_humidity', 
+                'atmospheric_pressure', 'etr', 'etrn', 'ghi_infrared', 'ghi', 
+                'dni', 'dhi', 'global_hor_illum', 'direct_normal_illum', 
+                'diffuse_horizontal_illum', 'zenith_luminance', 
+                'wind_direction', 'wind_speed', 'total_sky_cover', 
+                'opaque_sky_cover', 'visibility', 'ceiling_height', 
+                'present_weather_observation', 'present_weather_codes',
+                'precipitable_water', 'aerosol_optical_depth', 'snow_depth', 
+                'days_since_last_snowfall', 'albedo', 
+                'liquid_precipitation_depth', 'liquid_precipitation_quantity']
+   
+    # We only have to skip 6 rows instead of 7 because we have already used 
+    # the realine call above.
+    data = pd.read_csv(csvdata, skiprows=6, header=0, names=colnames)
+    
+    # Change to single year if requested
+    if coerce_year is not None:
+        data["year"] = coerce_year
+        data['year'].iloc[-1] = coerce_year - 1
+
+    # Update index with correct date information     
+    data = data.set_index(pd.to_datetime(data[['year', 'month', 'day', 
+                                               'hour']]))
+     
+    # Localize time series
+    data = data.tz_localize(int(meta['TZ'] * 3600))
```

Comment:
@wholmgren   I know this line is copied from `iotools.tmy` but how does it work?  I see that it does, but as far as I can tell, it's undocumented behavior of `pandas.DataFrame.tz_localize`.

Reply:
I don't know how, but we have a bit of context here: https://pvlib-python.readthedocs.io/en/latest/timetimezones.html#fixed-offsets

`data.tz_localize(pytz.FixedOffset(int(meta['TZ'] * 60)))` would be better. I have a vague recollection that it did not work on an earlier version of pandas, but probably ok now.

Reply:
Maybe a helper function to translate from timezone offset as a float to a `pytz` object. Suggest that we accept it in this PR, and open a new issue to improve here, in `iotools.tmy` and in readthedocs.